### PR TITLE
Shivani61 patch 11

### PR DESCRIPTION
**Pull request by shivani61**
Adding provisioner documentation

### CODE REVIEW

File: docs/home/provisioner.md [hunk 1]
```diff
@@ -0,0 +1,263 @@
+# Esper Device Provisioner
+<div class="avg-reading-time">Average reading time : {{ $page.readingTime.text.replace(' read', '') }}s</div>
```
Comment (bmehendale) [inline docs/home/provisioner.md]: @kirananto - what is the rationale of adding reading time to each page?

File: docs/home/provisioner.md [hunk 2]
```diff
@@ -0,0 +1,263 @@
...
+
+## Instructions for Windows 10 OS
+
+1. Download the Esper Device Provisioner from here:
+
+https://drive.google.com/open?id=1byC7jLxH18_m0YoW3k9NKif_FnZaog7N
```
Comment (bmehendale) [inline docs/home/provisioner.md]: This needs to be a downloads.esper.io/esper-provisioner/<something> link. @favazkandath - can you please confirm this here?

File: docs/home/provisioner.md [hunk 3]
```diff
@@ -0,0 +1,263 @@
+# Esper Device Provisioner
```
Comment (bmehendale) [inline docs/home/provisioner.md]: Need an introductory text.

File: docs/home/provisioner.md [hunk 4]
```diff
@@ -0,0 +1,263 @@
...
+<div class="avg-reading-time">Average reading time : {{ $page.readingTime.text.replace(' read', '') }}s</div>
+
+## Instructions for Windows 10 OS
```
Comment (bmehendale) [inline docs/home/provisioner.md]: (nit) remove `OS`

File: docs/home/provisioner.md [hunk 5]
```diff
@@ -0,0 +1,263 @@
...
+https://drive.google.com/open?id=1byC7jLxH18_m0YoW3k9NKif_FnZaog7N
+
+
+2. Open File Explorer (you can use the Windows Key + E shortcut), and go to where you have downloaded the file. Most probably, it will be in your Downloads’ folder:
+
+
+![](./assets/OLD_DASHBOARD/provisioner4.png)
+
+
+3. (Optional Step): Move the downloaded file to a temporary location, such as C:\temp\provisioning, so it’s easier to find the next time you need it:
```
Comment (bmehendale) [inline docs/home/provisioner.md]: I was wrong when I included step 3. Please remove it. The reason is that after installation, a shortcut icon is created in the start menu from which the provisioner can be launched. I thought this was not the case, and one needed to re-install each time.

File: docs/home/provisioner.md (hunk 1 above)
Comment (kirananto) [reply docs/home/provisioner.md]: Just an feature.. It shows you average time you need to spend on reading the contents of the page.

File: docs/home/provisioner.md (hunk 2 above)
Comment (favazkandath) [inline docs/home/provisioner.md]: @shivani61 please update the url.
Windows: http://downloads.esper.io/esper-provisioner/windows/Esper%20Device%20Provisioner
%20Setup%201.0.0-beta.exe
Mac: http://downloads.esper.io/esper-provisioner/mac/Esper%20Device%20Provisioner-1.0.0
-beta.dmg

File: docs/home/provisioner.md [hunk 6]
```diff
@@ -0,0 +1,263 @@
...
+
+![](./assets/OLD_DASHBOARD/provisioner17.png)
```
Comment (bmehendale) [inline docs/home/provisioner.md]: ok for now, but descriptive file names are recommended. eg. `prov_win10_dload_dir_screenshot.png`

File: docs/home/provisioner.md [hunk 7]
```diff
@@ -0,0 +1,263 @@
...
+
+
+4. Executable files downloaded from the internet are rightly not trusted by Windows by default. As a result, you have to tell Windows that it is safe to execute the file, as you’ve downloaded it from Esper, which you trust. To do so, right-click on the file, and click on Properties:
```
Comment (bmehendale) [inline docs/home/provisioner.md]: my bad. remove `, as you’ve downloaded it from Esper, which you trust.`

File: docs/home/provisioner.md [hunk 8]
```diff
@@ -0,0 +1,263 @@
...
+Apps that appear under the Uploaded Apps section will be downloaded locally to your PC and installed to your device in the next step. This makes the device skip the step to fetch the app from your endpoint during provisioning. This is especially helpful when you have multiple devices to provision, and want to save on the device’s bandwidth.
+
+Some devices require that some apps be side-loaded in addition to the ones listed in the template. These apps can be listed in the Additional APKs section. You can list the paths either by their remote URL or take then from your local computer.
+
+![](./assets/OLD_DASHBOARD/prov4.png)
+
+Note: Additional APKs always get installed before Uploaded Apps onto the device.
+
+
+13. Before you click Next on the above dialog, ensure that:
```
Comment (bmehendale) [inline docs/home/provisioner.md]: @favazkandath - you need to change this in the google doc. After your code change, point 13 here had significantly changed.

File: docs/home/provisioner.md [hunk 9]
```diff
@@ -0,0 +1,263 @@
...
+
+c. When you attach your device to the PC, you will be greeted with a prompt like the one shown below. Make sure you check the Always allow from this computer checkbox, followed by the OK button.
+
+![](./assets/OLD_DASHBOARD/prov6.png)
+
+
+d. Make sure you don’t have any accounts under Accounts in Settings.
+
+
+14. Once you do the above, click Next on the provisioning tool, and you will start seeing your device in it:
```
Comment (bmehendale) [inline docs/home/provisioner.md]: we need to talk about the special settings that  we can set via the toggle buttons.

File: docs/home/provisioner.md (hunk 2 above)
Comment (favazkandath) [reply docs/home/provisioner.md]: I've uodated the url with esper downloads url.

File: docs/home/provisioner.md (hunk 8 above)
Comment (favazkandath) [reply docs/home/provisioner.md]: Will do that @bmehendale

Comment (favazkandath) [reply docs/home/provisioner.md]: But @bmehendale we need to connect the device anyway to restart the adb daemon in tcpip mide right.

**Pull request merged**